CHANGES.md

```diff
@@ -37,6 +37,7 @@ Indexer:
 -   Fixed indexer incorrect parsing bounding box data in spatialCoverage aspect
 -   Added auth when crawling the registry
 -   Fixed Data.json spatial bounding box ordering not understood
+-   Indexer fails with "Object is missing required member 'id'"
```

Comment:
Maybe word this change as:
```suggestion
-   Handle and log the exception "Object is missing required member 'id'"
```

Reply:
Thanks~ That reads better~ Will update~ 👍